site/content/docs/5.3/components/pagination.md

```diff
@@ -30,8 +30,8 @@ Looking to use an icon or symbol in place of text for some pagination links? Be
 {{< example >}}
 <nav aria-label="Page navigation example with icons">
   <ul class="pagination">
-    <li class="page-item"><a class="page-link" href="#" aria-label="Previous"></a></li>
-    <li class="page-item"><a class="page-link" href="#" aria-label="Next"></a></li>
+    <li class="page-item"><a class="page-link" href="#"><span class="visually-hidden">Previous</span></a></li>
+    <li class="page-item"><a class="page-link" href="#"><span class="visually-hidden">Next</span></a></li>
```

Comment:
Change all occurence in the doc as well!
Do we add a migration note about this ?

Reply:
Note sure for the migration note since this PR is no more than an 'simple' a11y fix for https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1931 🤔

Reply:
Maybe just a quick note to remind people that they should change for better a11y ?